lib/dependency-manager-adapters/bower.js

```diff
@@ -172,26 +175,13 @@ module.exports = CoreObject.extend({
   },
   _findBowerPath: function() {
     var adapter = this;
-    return findEmberPath(adapter.cwd)
-      .then(function(emberPath) {
-        /* Find bower's entry point module relative to
-         ember-cli's entry point script */
-        return adapter._resolveModule('bower', { basedir: path.dirname(emberPath) })
-          .catch(function() {
-            debug('Bower not found');
-            return adapter._installBower();
-          }).then(function() {
-            return adapter._resolveModule('bower', { basedir: path.dirname(emberPath) });
-          });
-      })
-      .then(function(bowerPath) {
-        return '"' + path.join(bowerPath, '..', '..', 'bin', 'bower') + '"';
-      });
-  },
-  _installBower: function() {
-    var adapter = this;
-    debug('Installing bower via npm');
-    return adapter.run('npm', [].concat(['install', 'bower@^1.3.12']), { cwd: adapter.cwd });
+    return findEmberPath(adapter.cwd).then(function(emberPath) {
```

Comment:
I know you didn't add this, but I wonder if finding bower relative to ember-cli is necessary anymore?

Reply:
@kategengler hmm good point.  So presumably the logic would be

1. run npm before bower
2. trust that bower exists in `PROJECT_ROOT/node_modules/.bin/bower` and assert otherwise

Reply:
Yeah, or we can just leave this since it works, but just thought it worth thinking about

Reply:
@kategengler leaving it in this PR seems like the right thing to do; it'll be easier to track & revert in a separate PR if it causes issues (although I think you're right that finding it relative to ember-cli would no longer be useful).